desktop/apps/auction2/client/index.js

```diff
@@ -11,60 +11,62 @@ import { setupCommercialFilter } from './commercial_filter'
 import { data as sd } from 'sharify'
 import _ from 'underscore'
 
-const auction = new Auction(sd.AUCTION)
-const user = sd.CURRENT_USER ? new CurrentUser(sd.CURRENT_USER) : null
+export default () => {
```

Comment:
This diff is a little weird, but all I did was explicitly export the code here so we could require it in the assets file.